beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -990,14 +1090,51 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         Ok(())
     }
 
+    fn record_data_column_request_in_span(
+        &self,
+        peer_id: &PeerId,
+        requested_indices: &[ColumnIndex],
+        epoch_opt: Option<Epoch>,
+        span: Span,
+    ) {
+        let non_custody_indices = {
+            let custody_columns = self
+                .chain
+                .data_availability_checker
+                .custody_context()
+                .custody_columns_for_epoch(epoch_opt, &self.chain.spec);
+            requested_indices
+                .iter()
+                .filter(|subnet_id| !custody_columns.contains(subnet_id))
+                .collect::<Vec<_>>()
+        };
+        span.record("non_custody_indices", field::debug(non_custody_indices));
```

Comment:
This field is used to identify if peers are sending requests on columns we don't custody.